codalab/rest/bundles.py

```diff
@@ -426,6 +428,48 @@ def _fetch_locations():
     return dict(data=uuids_to_locations)
 
 
+@get('/bundles/<bundle_uuid:re:%s>/locations/', apply=AuthenticatedProtectedPlugin())
+def _fetch_bundle_locations(bundle_uuid: str):
+    """
+    Returns a list of BundleLocations associated with the given bundle
+
+    Query parameters:
+    - `bundle_uuid`: Bundle UUID to get the locations for
+    """
+    bundle_locations = local.model.get_bundle_locations(bundle_uuid)
+    return BundleLocationListSchema(many=True).dump(bundle_locations).data
+
+
+@post('/bundles/<bundle_uuid:re:%s>/locations/', apply=AuthenticatedProtectedPlugin())
+def _add_bundle_location(bundle_uuid: str):
```

Comment:
the final thing you should do is actually test these REST endpoints to make sure they work. You can run `python codalab_service.py start -bd` and then manually call the endpoints using postman -- see https://www.postman.com/ and https://buttercms.com/blog/postman-testing-the-api-development-environment

Reply:
Or if you don't want to use postman, you could instead create an automated test under `rest` in test_cli.py: https://github.com/codalab/codalab-worksheets/blob/master/tests/cli/test_cli.py#L2394. postman might be easier / quicker to do though